PSICover/PSICover.cs

```diff
@@ -150,6 +150,7 @@ class Analyzer {
    void GenerateOutputs () {
       ulong[] hits = File.ReadAllLines ($"{Dir}/hits.txt").Select (ulong.Parse).ToArray ();
       var files = mBlocks.Select (a => a.File).Distinct ().ToArray ();
+      var data = new List<(string, int, int, string)> ();

```

Comment:
Would be a little more clear if you use names:
`data = new List<(string File, int Total, int Hit, string HTML)> ();`
and in the rest of the code also use these names instead of Item1, Item2.